themes/bootstrap3/templates/combined/results.phtml

```diff
@@ -60,6 +60,9 @@
       <?php endforeach; ?>
     </div>
   <?php endif; ?>
+  <?php if ($config['Layout']['jump_links'] ?? false): ?>
+    <?=$this->context($this)->renderInContext('combined/jump-links.phtml', ['combinedResults' => $this->combinedResults])?>
```

Comment:
Do you need to use renderInContext here, or would a plain render work? (The renderInContext helper is a weird VuFind-specific thing that I'm hoping to someday eliminate... if regular render works, which I think it should in this case, it is preferable).

Reply:
@demiankatz Hmm.  Just tried this, a plan render() failed due to an undefined $columns param used by stack-??.phtml.  I fixed that by adding it to the $viewParams array above, but now I'm not clear if there are any other context variables that might be expected in certain conditions in any of the stack-* files.  I think it's safer to leave as is but your call.

Reply:
That happened after you changed `<?=$this->context($this)->renderInContext('combined/jump-links.phtml', ...` to `<?=$this->render('combined/jump-links.phtml', ...`? I wouldn't expect that, since as far as I can tell, jump-links.phtml only refers to one variable and doesn't include any sub-templates. But either I'm missing something, or we're not talking about the same code change. I wasn't suggesting changing any other renderInContext calls, just the newly-added jump-links one.

Reply:
Argh you are correct of course, I changed the wrong one.  I hate days I have to wake up at 5am :)

Reply:
Ha, I know exactly how you feel -- I was also up pretty early today, and I'm dragging at this point. :-)